core/src/main/java/nl/nn/adapterframework/xml/AttributesWrapper.java

```diff
@@ -51,6 +59,24 @@ public AttributesWrapper(Attributes source, String localNameToSkip) {
 				attributes.add(a);
 			}
 		}
+
+		if(sortAttributeOrder) {
+			attributes.sort(new Comparator<Attribute>() {
+				@Override
+				public int compare(Attribute o1, Attribute o2) {
+					String o1Name = o1.localName;
+					if ("".equals(o1Name)) {
+						o1Name = o1.qName;
+					}
+	
+					String o2Name = o2.localName;
+					if ("".equals(o2Name)) {
+						o2Name = o2.qName;
+					}
+					return o1Name.compareTo(o2Name);
+				}
```

Comment:
Ik denk dat je niet moet sorteren op qName, maar op uri en localname.
Als de prefix verschilt maar de namespace uri hetzelfde is, is de XML effectief dezelfde